core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7146,12 +7148,46 @@ components:
           $ref: "#/components/schemas/LedgerTransaction"
         receipt:
           $ref: "#/components/schemas/TransactionReceipt"
+        balance_changes:
+          $ref: "#/components/schemas/CommittedTransactionBalanceChanges"
         proposer_timestamp_ms:
           type: integer
           format: int64
           minimum: 0
           maximum: 100000000000000
           description: An integer between `0` and `10^14`, marking the proposer timestamp in ms.
+    CommittedTransactionBalanceChanges:
+      description: |
+        For the given transaction, contains individual entity resource balance changes.
```

Comment:
A lot of this isn't right - can we change this to:
```
The fee-related and non-fee-related balance changes of a transaction, aggregated by resource and global entity within which the change occurred.
```

Reply:
1. Do we want to keep CommittedTransactionBalanceChanges and existing LtsEntityFungibleBalanceChanges descriptions identical? I believe they should be.
2. What you proposed seems like a valid description for `CommittedTransactionBalanceChanges.fungible_entity_balance_changes` not the root type itself.

Reply:
1. Sounds sensible :+1:
2. I still think it's useful to highlight it on that type, to hint they need to look inside the `fungible_entity_balance_changes`

Reply:
Changed description / dropped resultant account balances